tests/apiv2/entities.py

```diff
@@ -66,6 +70,33 @@ def test_search_entities_subfields(self):
         self.assertEqual(data["entities"][0]["name"], "ta1")
         self.assertEqual(data["entities"][0]["type"], "threat-actor")
 
+    def test_search_entities_with_creation_date(self):
+        response = client.post(
+            "/api/v2/entities/search",
+            json={
+                "query": {"created": "<2020-01-02"},
+                "type": "threat-actor",
+            },
+        )
+        data = response.json()
+        self.assertEqual(response.status_code, 200, data)
+        self.assertEqual(len(data["entities"]), 1)
+        self.assertEqual(data["entities"][0]["name"], "ta1")
+        self.assertEqual(data["entities"][0]["type"], "threat-actor")
+
+        response = client.post(
+            "/api/v2/entities/search",
+            json={
+                "query": {"created": ">2020-01-03"},
```

Comment:
How can you assert based on created query while it is not set in entity2 creation?

Reply:
entity2 will be crated with a default of now(), so it should be caught here. I'll add a comment to the test so it's clearer. Thanks!

Reply:
entity2 will be crated with a default of now(), so it should be caught here. I'll add a comment to the test so it's clearer. Thanks!